### ISSUE TITLE
Karts seem to collide with road surface

### ISSUE THREAD

**Issue by asciimonster** [supertuxkart/stk-code#2002]
The karts in 0.8.2 seem to collide with the road surface. Without any karts present, the collision sound is heard. For instance it is specifically at these two places in the Scotland track:
![scotland-collision 1](https://cloud.githubusercontent.com/assets/10237862/6336426/c590c2c0-bba2-11e4-8da0-212d49a7a557.png)
![scotland-collision 2](https://cloud.githubusercontent.com/assets/10237862/6336427/c59406ba-bba2-11e4-9855-5af03f9537eb.png)

**Comment by tomreyn** [comment]
I've seen this happening, too. Even to a degree where my kart would overturn (there, too, no other kart was nearby). But I don't remember the tracks exactly, not the karts (it seems to happen more regularly with some karts, and with some tracks).

**Comment by Arthur-D** [comment]
I got a tip from @hiker to try lowering the downward-impulse-factor which is by default set to 5 to improve tracks with twisty roads like the newton test track. Setting it to 3 I saw a few warnings about karts being too high above ground in newton, but I figure there were so few occurrences a more carefully constructed track would not have this issue.

Testing in Scotland, I still hear the collision sound in the location in the second screenshot, but didn't really see any slowdown, and it is really a mesh problem in my opinion, so having downward-impulse-factor set to 3 seems to be an okay compromise - in the most extreme situations, there might be places where you lose ground contact, or hit the ground, but with well constructed tracks there should be no issues. Feel free to comment and test some more.
